ember-toucan-core/src/components/form/checkbox-field.ts

```diff
@@ -58,4 +80,16 @@ export default class ToucanFormCheckboxFieldComponent extends Component<ToucanFo
     assert('A "@label" argument is required', args.label);
     super(owner, args);
   }
+
+  get isChecked() {
+    if (!this.args?.option) {
+      return this.args.value;
+    }
```

Comment:
This essentially controls the case where there's a single checkbox in use.  Single checkboxes do not use `@option`, so we only need to respect `@value`.

Reply:
Ahhh....okay I'm sort of getting this now. 

Suggestion: I think the checkbox-field docs could be updated to reflect this.

Reply:
Is there a way to prevent the user from passing `@option` if the parent is not a checkbox field? 

(I don't actually know if there is a way to do this)

Reply:
So I was playing with discriminate unions in TS as I've used those before, but I couldn't quite get what I wanted with how the contextual components work. I think I may be able to use some asserts though to help prompt people to use the correct arguments. Thanks for the suggestion! I'll get something up tomorrow AM

Reply:
Okay so still no luck with TS, but I did add an assertion that essentially ensures folks either provide `@option` or `@value`, but not **both**.

https://github.com/CrowdStrike/ember-toucan-core/pull/81/commits/78e90b8a56783dfbbf6d45256c95cec9bc2b4407

Let me know what you think!

Reply:
This looks great!